manim/mobject/graph.py

```diff
@@ -559,7 +559,7 @@ def add_vertices(
         ----------
 
         vertices
-            A list of hashable vertex identifiers.
+            A comma separated list of hashable vertex identifiers.
```

Comment:
I think it is best to avoid using the word list to avoid confusion if it doesn't accept a list, and "comma separated" is already implicit in varargs, so maybe just "hashable vertex identifiers" is enough ?

Reply:
Done